infrastructure/constructs/lambda_endpoint.py

```diff
@@ -31,7 +32,7 @@ def __init__(
             handler=f"{BACKEND_DIRECTORY}.{package_name}.entrypoint.lambda_handler",
             runtime=PYTHON_RUNTIME,
             timeout=DEFAULT_LAMBDA_TIMEOUT,
-            code=bundled_code(package_name),
+            code=bundled_code(package_name, PACKAGING_DIR),
```

Comment:
Passing a constant around is usually unnecessary. Normally you'd just import a constant where it's used and reference it directly.

Reply:
Good spotting. This was a hang over from previous attempt. Have fixed.

Reply:
This also applies to `pip_install_requirements`.